build_tools/aws-sdk-code-generator/templates/errors_module.mustache

```diff
@@ -27,15 +27,28 @@ module {{module_name}}
   {{=<% %>=}}
   # * {<% name %>}
   <%={{ }}=%>
+  {{#query_protocol?}}
+  {{#custom_error_class?}}
+  #    * `{{custom_error_class}}` may be generated to use instead of this class name.
```

Comment:
We should use stronger language. The SDK WILL parse into a different error name and NOT use this error. We could even consider adding a deprecated yard trait to it.